examples/lighting-app/qpg/src/LightingManager.cpp

```diff
@@ -63,6 +84,14 @@ bool LightingManager::InitiateAction(Action_t aAction, int32_t aActor, uint16_t
     case LEVEL_ACTION:
         ChipLogProgress(NotSpecified, "LightMgr:LEVEL: lev:%u->%u", mLevel, *value);
         break;
+    case COLOR_ACTION_XY:
+        xy = *static_cast<XyColor_t *>(static_cast<void *>(value));
```

Comment:
Should we use an endianreader here and below for readability purposes?

Reply:
I think reinterpret_cast should do here - a similar conversion is done on the other side when sending the value to the callback.

Reply:
Actually the observation above applies to your remark in ZclCallbacks.cpp. For this case, I think since it will be compiled in the same endianness on both sides (sending in ZclCallbacks and reception in LightingManager), it will be endian safe.